src/__experimental__/components/radio-button/radio-button.style.js

```diff
@@ -91,10 +82,6 @@ const RadioButtonStyle = styled(CheckboxStyle)`
         padding: 4px 0;
```

Comment:
I think this will override any padding applied via the `labelSpacing` prop

Reply:
Yeah it will thanks, missed that